app/elements/editor-toolbar/editor-toolbar.html

```diff
@@ -16,73 +16,57 @@
         user-select: none;
       }
 
-      .main-toolbar.tall .app-name {
-        font-size: 40px;
-        font-weight: 300;
-        -webkit-transform-origin: left center;
-        transform-origin: left center;
+      .main-toolbar {
+        z-index: 10;
       }
 
-      .main-toolbar .middle-container,
-      .main-toolbar .bottom-container {
-        @apply(--layout-horizontal);
-        @apply(--layout-center);
-      }
-
-      .main-toolbar .middle-container  {
-        height: 100%;
-        margin-left: 48px;
-      }
-
-      .main-toolbar:not(.tall) .middle-container {
-        font-size: 18px;
-        padding-bottom: 0;
-      }
-
-      .main-toolbar .bottom-container {
-        @apply(--paper-font-subhead);
-
-        margin-left: 48px;
-        -webkit-transform-origin: left center;
-        transform-origin: left center;
+      .navbar-project-title {
+        --paper-input-container-input-color: #FFFFFF;
+        --paper-input-container-color: #FF5B81;
+        --paper-input-container-focus-color: #FF5B81;
+        width: 70%;
+        margin-left: 15px;
       }
     </style>
 
-    <paper-toolbar class="main-toolbar" style="z-index: 10;">
+    <paper-toolbar class="main-toolbar">
       <template is="dom-if" if="{{isNotEmpty(tools)}}">
         <paper-icon-button icon="menu" paper-drawer-toggle on-click="togglePanel"></paper-icon-button>
       </template>
 
-      <span class="space"></span>
-
-      <div class="middle middle-container" style="width: 100%;">
-        <!-- Application name -->
-        <img class="app-logo" src="../../images/logo.png" />
-        <div class="app-name" style="width: 90%">Gobstones</div>
-
-        <template is="dom-if" if="{{isNotEmpty(tools)}}">
-          <div class="buttons">
-            <template is="dom-if" if="{{showZoom(tools)}}">
-              <paper-button on-click="increaseEditorFontSize"><iron-icon class="button" icon="icons:zoom-in"></iron-icon></paper-button>
-              <paper-button on-click="decreaseEditorFontSize"><iron-icon class="button" icon="icons:zoom-out"></iron-icon></paper-button>
-            </template>
-            <template is="dom-if" if="{{permissions.can_use_library}}">
-              <paper-button on-click="toggleLibrary"><iron-icon class$="button {{buttonCssClass(showLibrary)}}" icon="icons:chrome-reader-mode"></iron-icon></paper-button>
-            </template>
-            <paper-button on-click="toggleShowBoards"><iron-icon class="button" icon$="icons:chevron-{{_showBoardsDirection(showBoards)}}"></iron-icon></paper-button>
-          </div>
-        </template>
-      </div>
+      <img class="app-logo" src="../../images/logo.png" />
+      <paper-input value="{{projectName}}" class="navbar-project-title" no-label-float>
```

Comment:
Todo esto debería estar entre `<template is="dom-if" if="{{isNotEmpty(tools)}}">` y `</template>` para que no se vea en la pantalla inicial. Y si querés cambiale el nombre a algo mejor como `areToolsVisible` o algo así :P